diagnostics/model_test/verify_model.py

```diff
@@ -2,14 +2,39 @@
 import argparse
 import numpy as np
 import ngraph
+from google.protobuf import text_format
 import json
 import os
 
 
+def createFolder(directory):
+    try:
+        if not os.path.exists(directory):
+            os.makedirs(directory)
+    except OSError:
+        print('Error: Creating directory. ' + directory)
+
+
+def set_os_env(select_device):
+    if select_device == 'CPU':
+        # run on TF only
+        ngraph.disable()
+    else:
+        if not ngraph.is_enabled():
+            ngraph.enable()
+
+        assert select_device[:
+                             7] == "NGRAPH_", "Expecting device name to start with NGRAPH_"
+        back_end = select_device.split("NGRAPH_")
+        os.environ['NGRAPH_TF_BACKEND'] = back_end[1]
+        # quantized mode:1 / float mode:0
+        os.environ['NNPI_CONVERT_FLOAT_TO_QUANT_GRAPH'] = str(quantized_mode)
```

Comment:
I'm wondering if this should be removed, but we setup something specifically, Not sure if this is right place to have NNPI specific changes go in.

Reply:
Good point

Reply:
Ok. make sense. Let me remove all the code related quantization.